examples/observability_archive/outputs.tf

```diff
@@ -10,4 +10,5 @@ output "log_analysis_name" {
 output "cos_bucket" {
   value       = module.cos
   description = "Cloud Object Storage information"
+  sensitive   = true
```

Comment:
Didn't we remove the sensitive values from this output?

Reply:
No, this is processing the raw output from the COS root module which will have any requested resource keys. This is where the keys are created and they have to be output.
Example code is included on how to remove the sensitive value if required and we removed it from the map of values returned (output) when using the buckets submodule so that the whole map is not hidden.

The choice here would be to duplicate the removal of the sensitive value in this example. Or to mark it sensitive.

Reply:
ah ok this is using the root level module and not the buckets submodule. I guess its fine for this example then